### PR TITLE
Add DataClearFilters docs

### PR DESCRIPTION
**Pull request by taysea**
Adds docs for DataClearFilters (new component in this PR: https://github.com/grommet/grommet/issues/6660)

### CODE REVIEW

File: src/screens/DataClearFilters.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      }}
+    />
+  </Page>
+);
+
+export default DataClearFilters;
+
+export const DataClearFiltersItem = ({ name, path }) => (
+  <Item name={name} path={path} center>
+    <Data data={[]}>{/* <DataClearFilters /> */}</Data>
```
Comment (taysea) [inline src/screens/DataClearFilters.js:40]:
```suggestion
    <Data data={[]}><DataClearFilters /></Data>
```

File: src/screens/DataClearFilters.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  Data,
+  // DataClearFilters
```
Comment (jcfilben) [inline src/screens/DataClearFilters.js:4]: Can we uncomment this and update yarn.lock to get the latest stable version of grommet?

File: src/screens/DataClearFilters.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+>
+  <Toolbar><DataClearFilters /></Toolbar>
+  <DataTable />
+</Data>`}
+      isA={{
+        base: 'Button',
+        path: '/button',
+      }}
+    />
+  </Page>
```
Comment (jcfilben) [inline src/screens/DataClearFilters.js:33]: In the theme section of the docs should we document `theme.data.button.kind`?

File: src/screens/DataClearFilters.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import Page from '../components/Page';
+import Item from './Components/Item';
+import { ComponentDoc } from '../components/Doc';
+
+const DataClearFiltersPage = () => (
+  <Page>
+    <ComponentDoc
+      name="DataClearFilters"
+      stable
+      availableAt={[
```
Comment (jcfilben) [inline src/screens/DataClearFilters.js:12]: Should we also link the storybook example here?
https://storybook.grommet.io/?path=/story/data-dataclearfilters-simple--simple

File: src/screens/Data.js [hunk 5]
```diff
@@ -299,6 +289,18 @@ const DataPage = () => (
         </Property>
       </Properties>
     </ComponentDoc>
+    <ThemeDoc>
```
[the remark this answers is not in the document]
Comment (taysea) [reply src/screens/Data.js]: Fixed.

File: src/screens/DataClearFilters.js (hunk 4 above)
Comment (taysea) [reply src/screens/DataClearFilters.js:12]: Fixed.

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**